pyworkflow/pyworkflow/node.py

```diff
@@ -11,14 +12,16 @@ def __init__(self, node_info, options=None):
         self.node_key = node_info.get('node_key')
         self.data = node_info.get('data')
 
+        self.is_global = True if node_info.get('is_global') else False
```

Comment:
Very minor suggestion: `self.is_global = node_info.get('is_global') is True` (or make it a property)

Do we have a mechanism to designate a flow variable as global? If it's given as an option to be configured by the user, then this line will need to check `self.options`, right?